### PR TITLE
feat(command-bar-reflow): move command bar to right of nav

### PR DESCRIPTION
**Pull request by lisli1**
#### Description of changes

- Moves command bar to the the right of the test navigation bar, in the right panel
- Command bar does not scroll with the rest of the content in the right panel, while the target page hidden banner does
- Focus goes from settings gear to nav bar, and then to command bar 

Gif of Assessment Heading function test with command bar in new position with different zoom levels from 100% to 400%
command-bar-right-resizing

#### Pull request checklist

- [x] Addresses an existing issue: # 1739632
- [x] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [x] (UI changes only) Added screenshots/GIFs to description above
- [x] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/DetailsView/Styles/detailsview.scss [hunk 1]
```diff
@@ -417,7 +417,6 @@ div.insights-file-issue-details-dialog-container {
                 word-break: break-word;
             }
             width: 100%;
-            padding-bottom: 1px;
```
Comment (pownkel) [inline src/DetailsView/Styles/detailsview.scss:420]: Why is this removed?

Comment (lisli1) [reply src/DetailsView/Styles/detailsview.scss:420]: I added the padding to the details-view-body-content-container instead

File: src/DetailsView/details-view-body.tsx [hunk 2]
```diff
@@ -104,9 +112,25 @@ export class DetailsViewBody extends React.Component<DetailsViewBodyProps> {
             ...this.props,
         };
 
+        if (this.props.featureFlagStoreData[FeatureFlags.reflowUI]) {
```
[the remark this answers is not in the document]
Comment (lisli1) [reply src/DetailsView/details-view-body.tsx]: Updated

File: src/DetailsView/details-view-body.tsx [hunk 3]
```diff
@@ -104,9 +112,25 @@ export class DetailsViewBody extends React.Component<DetailsViewBodyProps> {
...
     }
 
+    private renderReflowCommandBar(): JSX.Element {
+        const { switcherNavConfiguration } = this.props;
+
+        const detailsViewCommandBarProps: DetailsViewCommandBarProps = {
+            ...this.props,
+        };
+
+        if (this.props.featureFlagStoreData[FeatureFlags.reflowUI]) {
```
[the remark this answers is not in the document]
Comment (lisli1) [reply src/DetailsView/details-view-body.tsx]: Updated
